py/src/ai/h2o/sparkling/ml/params/H2OTypeConverters.py

```diff
@@ -268,3 +268,25 @@ def convert(value):
                 raise TypeError("Invalid type.")
 
         return convert
+
+    @staticmethod
+    def toDictionaryWithFloatElements():
+        def convert(value):
+            if value is None:
+                raise TypeError("None is not allowed.")
+            elif isinstance(value, JavaObject):
+                keys = [k for k in value.keySet().toArray()]
+                map = {}
+                for k in keys:
+                    map[k] = [TypeConverters.toFloat(v) for v in value.get(k)]
+                return map
+            elif isinstance(value, dict):
+                keys = value.keys()
+                map = {}
+                for k in keys:
+                    map[k] = TypeConverters.toFloat(value[k])
+                return map
```

Comment:
in both case you can also generate dictionaries using comprehensions if you want, e.g.:
```
return {k: [TypeConverters.toFloat(v) for v in value.get(k)] 
             for k in in value.keySet().toArray()}
```

Reply:
@seb-h2o Good idea! I've tried to address it, please take a look ;-)

Reply:
I don't know the possibilities of JavaObject , but with `dict` you can compress even more:
```
{k: TypeConverters.toFloat(v) for k, v  in value.items()}
```

Reply:
with `value` being a `JavaObject`, it's likely (however I didn't check) that it would work this way:
```{k: TypeConverters.toFloat(value[k]) for k, v in value.__dict__.items()}```

Reply:
checked `JavaObject`: forget last one

Reply:
`JavaObject` actually represents `java.util.Map[String, Double]` that is converted to python, so the api of that object should be identical

Thanks for the hint with `{k: TypeConverters.toFloat(v) for k, v  in value.items()}`!